assets/dev/js/editor/elements/views/container.js

```diff
@@ -0,0 +1,138 @@
+const BaseElementView = require( 'elementor-elements/views/base' ),
+	ColumnEmptyView = require( 'elementor-elements/views/column-empty' );
+
+const ContainerView = BaseElementView.extend( {
```

Comment:
Its better to not create new classes with old syntax, but use ES6
```suggestion
class ContainerView extends BaseElementView
```

There are few examples available at the project.

Reply:
Can you show me an example file? I get millions of errors when trying that ^^